### ISSUE TITLE
Unclear about "data.frame" in rhandsontables

### ISSUE THREAD

**Issue by ToonTalk** [ox-it/LiveData_htmlwidgets#52]
> The data.frame storing our data can be provided to rhandsontable for display, note that values in the table can be updated freely.

data.frame isn't explained or used in the code sample. Also need to explain what happens to the updated values -- they change something in the client's memory only? Additional libraries or features needed to talk to some data base?

**Comment by charliejhadley** [comment]
Yes, agreed. It isn't clear from the data import description what the object is, should change the description both in the import section and in the rhandsontable.

Regarding the updating of the data, this is possible within a shiny app. I can receive the data both client and server side, need a good shiny app to demo this.
